lookout/style/format/tests/test_postprocess.py

```diff
@@ -82,12 +82,37 @@ def test_posprocess(self):
     def test_bad_and_good_quotes(self):
         self.edit_and_test("""var a = '"0"'; var c = "0";""",
                            {4: (CLS_DOUBLE_QUOTE), 5: (CLS_DOUBLE_QUOTE,),
-                            10: (CLS_SINGLE_QUOTE), 11: (CLS_SINGLE_QUOTE,)},
+                            10: (CLS_SINGLE_QUOTE), 11: (CLS_SINGLE_QUOTE)},
```

Comment:
I see that tuples were changed in the PR splitting quotes and other tokens but `_to_label` expects sequences of labels. It still works because technically a string is a sequence so even a character is a sequence but I'd rather keep the tuples. So basically:

                               {4: (CLS_DOUBLE_QUOTE,), 5: (CLS_DOUBLE_QUOTE,),
                                10: (CLS_SINGLE_QUOTE,), 11: (CLS_SINGLE_QUOTE,)},

Reply:
good catch! thanks.